cmd/gateway-unsupported.go

```diff
@@ -261,6 +261,11 @@ func (a GatewayUnsupported) IsEncryptionSupported() bool {
 	return false
 }
 
+// IsObjectTaggingSupported returns whether object tagging is supported or not for this layer.
+func (a GatewayUnsupported) IsObjectTaggingSupported() bool {
```

Comment:
Is there a plan to add bucket tagging support for gateways? If yes, you could have generic name like `IsTaggingSupported()`